src/api/java/baritone/api/Settings.java

```diff
@@ -1183,6 +1183,21 @@
      */
     public final Setting<Boolean> notificationOnMineFail = new Setting<>(true);
 
+    /**
+     * Disallow block placement and breaking within a range from the home waypoint(if it exists)
+     */
+    public final Setting<Double> homeProtectionRange = new Setting<>(0.0);
+
+    /**
+     * Sends a chat notification to let you know when you enter or leave protected home area, can be quite annoying when you're working close to a home waypoint
+     */
+    public final Setting<Boolean> homeProtectionChatNotification = new Setting<>(true);
```

Comment:
Why do we need chat notifications (and even a toggle setting for them) here? If you can see the chat message, you can see what Baritone is doing anyway. I think turning the chat notifications into desktop notifications would make more sense, but even then I'm not sure if we really need it.

Reply:
I thought it would be handy for people who have accidently turned the protection range up and are wondering why baritone "doesn't work"